spec/test_app/app/controllers/basechurch/v1/groups_controller_spec.rb

```diff
@@ -3,7 +3,10 @@
 describe Basechurch::V1::GroupsController, type: :controller do
   describe 'GET /groups/:id' do
     let(:group) do
-      create(:group, created_at: DateTime.iso8601('2001-02-03T04:05:06+07:00'))
+      create(:group,
+             created_at: DateTime.iso8601('2001-02-03T04:05:06+07:00'),
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.